projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/write-case-flag-field.component.ts

```diff
@@ -29,10 +29,13 @@ export class WriteCaseFlagFieldComponent extends AbstractFieldWriteComponent imp
   public flagCommentsOptional = false;
   public jurisdiction: string;
   public isDisplayContextParameterUpdate: boolean;
+  public isDisplayContextParameterExternal: boolean;
   public caseTitle: string;
   public caseTitleSubscription: Subscription;
   private allCaseFlagStagesCompleted = false;
   private readonly updateMode = '#ARGUMENT(UPDATE)';
+  private readonly createExternalMode = '#ARGUMENT(CREATE,EXTERNAL)';
+  private readonly updateExternalMode = '#ARGUMENT(CREATE,EXTERNAL)';
```

Comment:
@MateiHMCTS - I think the updateExternalMode should be #ARGUMENT(**UPDATE**,EXTERNAL). Please check with @Daniel-Lam and also update the test file if change is required.